src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -1231,4 +1233,62 @@ private static List<Map<String, Object>> sortByValue(final List<Map<String, Obje
                 .sorted(Comparator.comparing(map -> (String) map.get(key)))
                 .collect(Collectors.toList());
     }
+
+    @Test
+    @ExtendWith(FeatureToggleConditionExtension.class)
+    void updateContactInformationForOrganisationShouldReturnSuccess() {
+        log.info("updateContactInformationForOrganisationShouldReturnSuccess :: STARTED");
+
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        String organisationIdentifier = (String) response.get("organisationIdentifier");
+
+        assertThat(organisationIdentifier).isNotEmpty();
+        JsonPath orgResponse = professionalApiClient.retrieveOrganisationDetails(
+            organisationIdentifier, hmctsAdmin,OK);
+        assertNotNull(orgResponse);
+        List<HashMap>  existingContactInformationList =  (List) orgResponse.get("contactInformation");
+
+        assertThat(existingContactInformationList).isNotEmpty()
+            .hasSizeGreaterThan(0);
+
+        List<ContactInformationCreationRequest> contactInformationCreationRequestList =
+            professionalApiClient.createContactInformationCreationRequests();
```

Comment:
both existing addresses and new addresses are same, create new addresses with different values and assert.